lms/templates/courseware/course_updates.html

```diff
@@ -1,11 +1,10 @@
 <%! from django.utils.translation import ugettext as _ %>
 <section aria-labelledby="course-updates-heading">
-    <h2 class="hd hd-2 sr" id="course-updates-heading">${_('All course updates')}</h2>
```

Comment:
@luckyv as we discussed, there is already a visible heading here that is sufficient to structure the page, so the SR-only heading here is redundant.  @cahrens is correct in that now we have to remove the ```aria-labelledby``` in the section element above.  However, we have a rule here that section elements must have a heading associated with them.  Either make the visible heading a child of this section (which seems hard to do because its in another template) OR just change the ```section``` element above to a ```div``` then the rule doesn't apply.  I don't think a section element is necessary here since it is the primary content of the main element.  If this page were more complicated, perhaps section elements would be necessary, but even then I don't think so.  The HTML5 spec seems to support this:

> The section element represents a generic section of a document or application. A section, in this context, is a thematic grouping of content. The theme of each section should be identified, typically by including a heading (h1-h6 element) as a child of the section element.

> Examples of sections would be chapters, the various tabbed pages in a tabbed dialog box, or the numbered sections of a thesis. A Web site's home page could be split into sections for an introduction, news items, and contact information.

> NOTE: The section element is not a generic container element. When an element is needed only for styling purposes or as a convenience for scripting, authors are encouraged to use the div element instead. A general rule is that the section element is appropriate only if the element's contents would be listed explicitly in the document's outline.